lib/types.nix

```diff
@@ -196,6 +254,15 @@ rec {
       functor = (defaultFunctor name) // { wrapped = elemType; };
     };
 
+    # TODO: deprecate, like `list'
```

Comment:
`attrs` is not deprecated! Remove this miss-leading comment.

Reply:
In the removed doc commit. I hope I don’t forget to remove it when PR’ing that.